docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -295,13 +272,12 @@ add the `- ip_netmask: 172.17.3.60` line that you removed in the source node.
 $ sudo os-net-config -c /etc/os-net-config/config.yaml
 ----
 
-. Verify, using the `ip` command, that the IP address is present in the target
-node.
+. Verify that the IP address is present in the target node.
```

Comment:
Can we add an example of the `ip` command they would use?
E.g., `[controller-0]$ ip …` (I'm not sure about the rest).